habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/ProcessExecutor.java

```diff
@@ -187,7 +190,15 @@ public final void flush() {
 
         @Override
         protected void processLine(final String line, final int logLevel) {
-            logger.info(line);
+            if (line.toUpperCase().contains(ERROR_LOG) || line.toUpperCase().contains(CRITICAL_LOG)) {
```

Comment:
A: Not too much of a difference, but I think `startsWith` over `contains` makes this logic less susceptible to edge cases.

Reply:
Yea actually thats what I started with but I noticed that some of the aiSSEMBLE logs have the following format: `2024/03/20 13:06:51 INFO Environment: Starting Test container services`, where the level isn't listed at the beginning. So I switched to `contains`